### PR TITLE
Add foobar2000 1.5.4.0

### PR DESCRIPTION
**Pull request by suasgn**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/1016)

### CODE REVIEW

**Comment by boarnoah** [comment]
Hi this is going to run into the same issues as my prior PR for this program https://github.com/microsoft/winget-pkgs/pull/518#issuecomment-633364260

Namely the URLs are not permanent, they expire after a time period.

**Comment by superusercode** [comment]
why not try a mirror like https://www.majorgeeks.com/files/details/foobar2000.html ?

**Comment by suabahasa** [comment]
I'm update the url from videohelp. the hash is original from the official one

File: manifests/foobar2000/foobar2000/1.5.4.0.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Id: foobar2000.foobar2000
+Publisher: foobar2000
+Name: foobar2000
+Version: 1.5.4.0
+License: BSD
```
Comment (boarnoah) [inline manifests/foobar2000/foobar2000/1.5.4.0.yaml:5]: I'm fairly sure the foobar2000 app is not BSD licensed, the official website lists it as [foobar2000 license](https://www.foobar2000.org/license) which is a custom freeware thing.

Comment (suabahasa) [reply manifests/foobar2000/foobar2000/1.5.4.0.yaml:5]: what should I change it to? `Copyright © Peter Pawlowski` or `Freeware` ?

Comment (boarnoah) [reply manifests/foobar2000/foobar2000/1.5.4.0.yaml:5]: Copyright would be more accurate I think, you can also add the additional field LicenseUrl that points to the document. :)

**Comment by KevinLaMS** [comment]
I am not sure if the settings are correct.  When  i ran the installer, nothing seemed to install.

**Comment by superusercode** [comment]
@KevinLaMS according to https://silentinstallhq.com/foobar2000-silent-install-how-to-guide/ the silent install must be done in an elevated command prompt. Maybe that has to do with it not installing properly?

**Pull request closed without merging**